packages/fast-form-generator-react/README.md

```diff
@@ -0,0 +1,181 @@
+
+# ReactJS FAST form generator
+A self generating UI based on JSON Schemas.
+
+It can be utilized to dynamically generate a form for changing data values on a React component.
+
```

Comment:
Explanation isn't very clear to me.... would change to something like: "Dynamically generates a form user interface based on incoming JSON Schemas to change data values of a React component."

Reply:
fixed